Sources/TuistGenerator/Mappers/PodInstallProjectMapper.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+/// A mapper that returns the side effects for installing Pods in a given project..
+public class PodInstallProjectMapper: ProjectMapping {
+    public init() {}
+
+    public func map(project: Project) throws -> (Project, [SideEffectDescriptor]) {
+        var project = project
+        var podProjectPaths = project.podProjectPaths
+
+        let podProjectPathsToAdd = project.targets
```

Comment:
After moving away from CocoaPods as a target dependency this step will no longer be necessary, only the side effects part.

Reply:
+1 on removing it from dependencies